recipe/meta.yaml

```diff
@@ -80,9 +78,6 @@ test:
     - conda build --help
     - conda convert --help
     - conda develop --help
-    - conda inspect --help
-    - conda inspect linkages --help  # [unix]
-    - conda inspect objects --help  # [osx]
```

Comment:
Hm. This is coming from 94ea1ec4b06bdbc27915cbc6130f0d4653b8fd2a, which the bot definitely shouldn't have removed. I'll rebase on Bianca's commit directly...